beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -410,21 +410,13 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
     /* Error responses */
 
     pub fn peer_disconnected(&mut self, peer_id: &PeerId) {
+        /* Check disconnection for single lookups */
```

Comment:
```suggestion
        // Check disconnection for single lookups.
```
or was there a reason for the block comments?